cdapython.py

```diff
@@ -149,14 +149,14 @@ def _get_result(self, _offset, _limit):
 
 def columns(version=table_version, host=CDA_API_URL):
     """Get columns names from the database."""
-    query = f"SELECT field_path FROM `gdc-bq-sample.cda_mvp.INFORMATION_SCHEMA.COLUMN_FIELD_PATHS` WHERE table_name = '{version}'"
+    query = f"SELECT field_path FROM `gdc-bq-sample.cda_mvp.INFORMATION_SCHEMA.COLUMN_FIELD_PATHS`"
```

Comment:
Does it work without this line? I would expect the `table_name = ` clause to be required here.